### PR TITLE
Revert "Bugfix For Opening Projects With Missing Verse (#6343)"

### PR DESCRIPTION
**Pull request by RoyalSix**
This reverts commit cd1a0a812da56f8467ab44069a360100b77346d5.

#### Describe what your pull request addresses (Please include relevant issue numbers):
-

#### Please include detailed Test instructions for your pull request:
-

#### Standard Test Instructions for PR Review Process:

- [ ] Double check unit tests that have been written
- [ ] Check for documentation for code changes
- [ ] Check that there are not inadvertent commits to tC Apps when reviewing a tC Core PR
- [ ] Checkout the branch locally and ensure that app runs as expected
  - [ ] Ensure tests pass
  - [ ] Open and watch the console for errors
  - [ ] Make sure all actions perform as expected
  - [ ] Import and Load a new Project
  - [ ] Load a tool and perform basic actions
  - [ ] Switch tools and perform basic actions
  - [ ] Switch project to an existing project
  - [ ] Load a tool and perform basic actions
  - [ ] Switch tools and perform basic actions
  - [ ] Next time reverse the order of importing after loading an existing project
- [ ] Reviewer should double check the DoD in the ISSUE, including the “spirit” of the story
- [ ] Ask Ben or Koz if you have any concerns about implementation (especially UI related)

---
This change is [Reviewable](https://reviewable.io/reviews/unfoldingword-dev/translationcore/6396)

### CODE REVIEW

**Comment by dsm-git** [comment]
[Linux build 2.0.0 (7c2b0ca)](https://tc-builds.door43.org/build/feature-jay-revert-2/translationCore-linux-x64-2.0.0-7c2b0ca.deb) is ready for download.
[macOS build 2.0.0 (7c2b0ca)](https://tc-builds.door43.org/build/feature-jay-revert-2/translationCore-macos-x64-2.0.0-7c2b0ca.dmg) is ready for download.
[Windows x64 build 2.0.0 (7c2b0ca)](https://tc-builds.door43.org/build/feature-jay-revert-2/translationCore-win-x64-2.0.0-7c2b0ca.setup.exe) is ready for download.
[Windows x32 build 2.0.0 (7c2b0ca)](https://tc-builds.door43.org/build/feature-jay-revert-2/translationCore-win-x32-2.0.0-7c2b0ca.setup.exe) is ready for download.

File: src/js/containers/projectValidation/ProjectValidationContainer.js [hunk 1]
```diff
@@ -24,10 +24,6 @@ import {withLocale} from '../Locale';
 class ProjectValidationContainer extends Component {
   shouldComponentUpdate(nextProps) {
     const {loadingProject} = nextProps.reducers.homeScreenReducer;
-    const { showProjectValidationStepper } = nextProps.reducers.projectValidationReducer;
-    if (showProjectValidationStepper === true) {
-      return true;
-    }
     if (loadingProject === true) {
```
Comment (mannycolon) [inline src/js/containers/projectValidation/ProjectValidationContainer.js]: Should `loadingProject` be removed as well?

Comment (RoyalSix) [reply src/js/containers/projectValidation/ProjectValidationContainer.js]: It looks like it is getting removed in the other PR.  https://github.com/unfoldingWord-dev/translationCore/pull/6397/files#diff-f413ef91441c3a0ad6e44e8adbdf8389
Whichever you want to merge first I will check for merge conflicts and make sure its all good.

Comment (mannycolon) [reply src/js/containers/projectValidation/ProjectValidationContainer.js]: Merge w/ latest dev branch

Comment (RoyalSix) [reply src/js/containers/projectValidation/ProjectValidationContainer.js]: Done.

**Comment by dsm-git** [comment]
[Linux build 2.0.0 (f43c2e8)](https://tc-builds.door43.org/build/feature-jay-revert-2/translationCore-linux-x64-2.0.0-f43c2e8.deb) is ready for download.
[macOS build 2.0.0 (f43c2e8)](https://tc-builds.door43.org/build/feature-jay-revert-2/translationCore-macos-x64-2.0.0-f43c2e8.dmg) is ready for download.
[Windows x64 build 2.0.0 (f43c2e8)](https://tc-builds.door43.org/build/feature-jay-revert-2/translationCore-win-x64-2.0.0-f43c2e8.setup.exe) is ready for download.
[Windows x32 build 2.0.0 (f43c2e8)](https://tc-builds.door43.org/build/feature-jay-revert-2/translationCore-win-x32-2.0.0-f43c2e8.setup.exe) is ready for download.

File: src/js/containers/projectValidation/ProjectValidationContainer.js (hunk 1 above)
Comment (mannycolon) [reply src/js/containers/projectValidation/ProjectValidationContainer.js]: @RoyalSix it still looks like is there?

**Review by mannycolon — APPROVED** [review]

**Comment by dsm-git** [comment]
[Linux build 2.0.0 (65ac90d)](https://tc-builds.door43.org/build/feature-jay-revert-2/translationCore-linux-x64-2.0.0-65ac90d.deb) is ready for download.
[macOS build 2.0.0 (65ac90d)](https://tc-builds.door43.org/build/feature-jay-revert-2/translationCore-macos-x64-2.0.0-65ac90d.dmg) is ready for download.
[Windows x64 build 2.0.0 (65ac90d)](https://tc-builds.door43.org/build/feature-jay-revert-2/translationCore-win-x64-2.0.0-65ac90d.setup.exe) is ready for download.
[Windows x32 build 2.0.0 (65ac90d)](https://tc-builds.door43.org/build/feature-jay-revert-2/translationCore-win-x32-2.0.0-65ac90d.setup.exe) is ready for download.

**Pull request merged**